modules/gapi/include/opencv2/gapi/gcommon.hpp

```diff
@@ -80,6 +80,7 @@ enum class GShape: int
     GSCALAR,
     GARRAY,
     GOPAQUE,
+    GFRAME,
```

Comment:
Make sense to handle it in corresponding `switch`

Reply:
it is already handled in `default:` which throws.

Reply:
haha, no. Added now